examples/bzlmod/__init__.py

```diff
@@ -0,0 +1,8 @@
+# TODO: bzlmod should grant access to pip_install dependencies as well
```

Comment:
Do we need to leave this TODO (not possible yet?) here or should we resolve before merging?

Reply:
I'd like to get this much in since it's feature parity for what bazel-central-registry currently provides (for an old version of rules_python)

I think it will take a bit of research and tinkering to see how pip deps can be exposed too.